helm/botkube/templates/service.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.service }}
+{{- range $port := .Values.service.ports  }}
+{{- $service_type := $port.type | default "ClusterIP" }}
```

Comment:
@Surbhidongaonkar We shouldn't be creating service if serviceMonitor.enabled is not set true

Reply:
Have modified service.yaml with required changes